KubeArmor/enforcer/appArmorEnforcer.go

```diff
@@ -124,14 +123,14 @@ func NewAppArmorEnforcer(node tp.Node, logger *fd.Feeder) *AppArmorEnforcer {
 	}
 
 	for _, file := range files {
-		if !file.Mode().IsRegular() {
+		if !file.Type().IsRegular() {
```

Comment:
So the return type was changed as well to DirEntry as well.
ref https://pkg.go.dev/os#DirEntry